frontend/src/scenes/persons/Person.component.spec.tsx

```diff
@@ -0,0 +1,957 @@
+import '~/mocks/window' // must be before Person
+import { Person } from 'scenes/persons/Person'
+import React from 'react'
+import { render, screen } from '@testing-library/react'
+import '@testing-library/jest-dom'
+import { defaultAPIMocks, mockAPI } from 'lib/api.mock'
+import { initKea } from '~/initKea'
+import { router } from 'kea-router'
+import { getContext } from 'kea'
+import { Provider } from 'react-redux'
+
+// TODO does this need `mockPosthog` from cypress/support/helpers.js
+jest.mock('posthog-js', () => ({
+    onFeatureFlags: jest.fn(),
+}))
+
+jest.mock('lib/api')
+
+describe('the person component', () => {
+    beforeEach(() => {
```

Comment:
For component tests it feels really weird to be mocking APIs and initializing kea - we're not really testing the component at that point, but rather the logic + kea + component.

Wouldn't it make more sense to mock the kea logic with static values and check how things render?

Reply:
Yeah, I wondered how much setup we'd need. It'd be shared between tests ultimately like in Cypress... But feels like, as it is now, if a test failed it would be hard to see why it had failed.

I'm wondering if visual regression tests would give the same confidence at lower cost

Reply:
Yep, much nicer to write if you restrict yourself to components with no kea logic